README.md

```diff
@@ -146,59 +146,75 @@ coastal area.
 
 ### MDWaterBox
 
-This scenario simulates a system that consists of a cubic box of water molecules, evolving according to the rules of Molecular Dynamics - the position of the water molecules is updated using Newton's equation in discrete time steps. The force that acts upon the particles is computed using standard molecular force fields. This approach is inspired by [this article](https://arxiv.org/abs/2112.03383).
+This scenario simulates a system that consists of a cubic box filled with water molecules, evolving according to the rules of Molecular Dynamics: this means that the water molecules' positions are updated according to Newton's equation in discrete time steps. The force that acts upon the particles is computed using standard molecular force fields. The implementation of this scenario was inspired by [this article](https://arxiv.org/abs/2112.03383). 
 
 #### Example
 
 First we initialize the scenario:
-```
+```python
 from inductiva.molecules.scenarios import MDWaterBox
 
 scenario = MDWaterBox(temperature = 300, box_size = 2.3)
 ```
 
-The user can specify the temperature (in Kelvin) and box size (length of one of the cube's edges, in nanometers). The numbers above correspond to the default values.
+The user can specify the temperature (in Kelvin) and the box size (length of one of the cube's edges, in nanometers). The numbers above correspond to the default values.
 
 After the initialization, we are ready to simulate the system:
 
-```
-output = scenario.simulate(simulation_time = 10,
-            nsteps_minim = 5000)
+```python
+output = scenario.simulate(simulation_time = 10, nsteps_minim = 5000)
 ```
 
-The simulate method initializes a simulation in the cloud. In this call, we set the parameters:
+The simulate method initializes a simulation using cloud resources. In this call, we set the following parameters:
  - simulation_time = 10: sets the trajectories time to span 10 **nanosecons**;
- -nsteps_minin = 5000: sets the number of minimization steps in the energy minimization step. 
+ - nsteps_minin = 5000: sets the number of minimization steps in the energy minimization step.
```

Comment:
I feel these defaults are weird here, but I will check over all.

Reply:
I will take this part out too, in the next PR.